chart/templates/operator.yaml

```diff
@@ -646,6 +646,18 @@ spec:
           capabilities:
             drop:
             - ALL
+      initContainers:
+      - args:
+        - |
```

Comment:
Why not include this arg along with the other one in `command`?

Reply:
I guess it is a preference, I like to write the script code on multiple lines, easier to copy&past for testing